### PR TITLE
MM-58183 Post Error Style Fixes

### PR DESCRIPTION
**Pull request by matthewbirtch**
#### Summary
Fixes alignment of error messages on posts and makes font sizes consistent with message typing.

#### Ticket Link
Jira https://mattermost.atlassian.net/browse/MM-58183

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by marianunez — APPROVED** [review]
Thanks @matthewbirtch! I was checking in the test server if the RHS also needed a fix for this but seems good there 👍

**Review by yasserfaraazkhan — APPROVED** [review]
image

image

**Comment by matthewbirtch** [comment]
@hmhealey could you review this one when you have a moment?

**Comment by hmhealey** [comment]
Yep, will do. Not sure how I missed this one

File: webapp/channels/src/components/advanced_text_editor/advanced_text_editor.scss [hunk 1]
```diff
@@ -181,20 +180,18 @@
             }
         }
 
-        &--has-error {
```
Comment (hmhealey) [inline webapp/channels/src/components/advanced_text_editor/advanced_text_editor.scss:184]: It looks like we're not using this class elsewhere, so would you mind removing that from [this line](https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx#L757) by changing it to `className='AdvancedTextEditor__footer'`?

**Review by hmhealey — APPROVED** [review]
These changes look good to me, but could you also remove the one place that the `&--has-error` class is used since the class no longer does anything?

**Pull request merged**